simulation/revolve2/simulation/scene/_multi_body_system.py

```diff
@@ -189,18 +189,21 @@ def calculate_aabb(self) -> tuple[Vector3, AABB]:
         points: list[Vector3] = []
         for rigid_body in self._rigid_bodies:
             for geometry in rigid_body.geometries:
-                if not isinstance(geometry, GeometryBox):
+                if isinstance(geometry, GeometryCylinder):
```

Comment:
since we use AABB here maybe make it part of the GeometryCilinder class directly, and make comment there what it is